src/metric-engine/src/engine/create.rs

```diff
@@ -51,13 +54,15 @@ impl MetricEngineInner {
         &self,
         region_id: RegionId,
         request: RegionCreateRequest,
+        extension_return_value: &mut HashMap<String, Vec<u8>>,
```

Comment:
Can we use return value? Put it into the params looks not good.

Reply:
This map will be overwritten in this method. So a mutable map reference is necessary.